docs/manual/constant-value-checker.tex

```diff
@@ -251,6 +251,20 @@ \section{Warnings\label{value-checker-warnings}}
 
 \end{itemize}
 \end{sloppypar}
+
+
+\section{Overflow\label{value-checker-overflow}}
+
+The Constant Value Checker normally takes Java's normal overflow rules into account when
+computing the possible values of expressions.
+The \code{-AignoreRangeOverflow} command-line option
+makes it ignore the possibility of overflow
+for range annotations (\<@IntRange>, for instance).
+Ignoring overflow is useful for some checkers
```

Comment:
I had a request on a previous iteration of this code review for an explanation of how a user decides whether to supply this flag.  If we cannot explain how a user should make that decision and what good it does, then it should not be user-visible option.  And, without that information, it's hard for me to review the code to assess whether it achieves that goal.